doc/classes/String.xml

```diff
@@ -887,6 +887,13 @@
 				[/codeblocks]
 			</description>
 		</method>
+		<method name="validate_node_name">
+			<return type="String">
+			</return>
+			<description>
+				Removes any characters from the string that are prohibited in Node names.
```

Comment:
```suggestion
				Removes any characters from the string that are prohibited in [Node] names.
```
I think we might as well give the list of those characters here.

Reply:
Done